CONTRIBUTING.md

```diff
@@ -1,5 +1,117 @@
 # Contributing to the pyOpenSci Python package template
 
+
+Contributions are welcome, and they are greatly appreciated! Every little bit
+helps, and credit will always be given.
+
+## Example Contributions
+
+You can contribute in many ways, for example:
+
+* [Report bugs](#report-bugs)
+* [Fix Bugs](#fix-bugs)
+* [Implement Features](#implement-features)
+* [Write Documentation](#write-documentation)
+* [Submit Feedback](#submit-feedback)
+
+### Report Bugs
+
+Report bugs at https://github.com//juno/issues.
```

Comment:
That doesnt seem right

Reply:
oh gosh. let me submit another PR with more fixes @sneakers-the-rat following your review and after merging your other PR. I was playing with making different packages using different usernames (including my dog Juno) and I copied the contributing file from the resulting package here - so we have a contributing file and then I created development guide that is separate.